src/hooks/useHomeAnimations.js

```
@@ -0,0 +1,211 @@
+          ease: "back.out(1.7)",
+        },
+        "-=0.4"
+      );
+
+    // Cleanup
+    return () => {
+      if (rafId) {
+        cancelAnimationFrame(rafId);
+      }
+      lenis.destroy();
+      ScrollTrigger.getAll().forEach((t) => t.kill());
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)